pkgs/applications/virtualization/docker/default.nix

```diff
@@ -144,6 +144,9 @@ rec {
     pname = "docker";
     inherit version;
 
+    # below inherits allow overriding of docker components
+    inherit docker-runc docker-containerd docker-proxy docker-tini moby;
```

Comment:
```suggestion
    inherit moby;
```
I can only see moby being used in this function. How are the other ones used?

Reply:
you're right, seems that i missed that; i'll change it to only inherit moby.